src/utilities/store.ts

```diff
@@ -30,7 +31,12 @@ const middlewares = [
   // reduxLogger,
 ].filter((middleware) => typeof middleware === 'function');
 
-const registry = new ReducerRegistry({}, middlewares);
+const composeEnhancers =
+  typeof window === 'object' && (window as any).__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ && window.location.hostname.includes('.foo.redhat.com')
+    ? (window as any).__REDUX_DEVTOOLS_EXTENSION_COMPOSE__({})
+    : compose;
+
+const registry = new ReducerRegistry({}, middlewares, composeEnhancers);
```

Comment:
@Hyperkid123 @karelhala any cons in doing this? I don't see how we can use redux without the devtools